core/src/test/java/com/novoda/noplayer/exoplayer/ExoPlayerTwoImplTest.java

```diff
@@ -527,4 +545,13 @@ private MediaSource givenMediaSource() {
     private void givenPlayerIsAttached() {
         player.attach(playerView);
     }
+
+    private SurfaceHolder whenSurfaceHolderIsReady() {
+        ArgumentCaptor<SurfaceHolderRequester.Callback> argumentCaptor = ArgumentCaptor.forClass(SurfaceHolderRequester.Callback.class);
+        verify(surfaceHolderRequester).requestSurfaceHolder(argumentCaptor.capture());
+        SurfaceHolderRequester.Callback callback = argumentCaptor.getValue();
+        SurfaceHolder surfaceHolder = mock(SurfaceHolder.class);
+        callback.onSurfaceHolderReady(surfaceHolder);
```

Comment:
Isn't this logic just repeating what we do in [the `setUp()`](https://github.com/novoda/no-player/blob/7c7215ec420a14819bda9c2846a27e37f6464e97/core/src/test/java/com/novoda/noplayer/exoplayer/ExoPlayerTwoImplTest.java#L140-L146)?